### PR TITLE
editor toggle now works

### PR DESCRIPTION
**Pull request by sdellis**

### CODE REVIEW

**Comment by aeschylus** [comment]
I think this one can be merged. Looks good and creates no bugs that I can see testing it locally. Let's write some tests for this soon.

**Comment by aeschylus** [comment]
Add more commits to this one in order to reopen it. Comments in code.

**Comment by aeschylus** [comment]
[here](https://github.com/IIIF/mirador/pull/633/files#diff-1bb2d28769af970c5242c3925aa2ca0cR86) is where we should update the list. The state update should call a handler that compares the page data and updates the local state if it has changed. 

Alternatively, (and more granularly) the editorPanel can listen for the page turn event and set itself to a blank/ "loading" template, and then when the list is loaded, re-populate in response to the state update event from annotationsTab.

File: js/src/widgets/editorPanel.js [hunk 1]
```diff
@@ -92,6 +97,7 @@
             jQuery.subscribe('openAnnotationList.' + this.windowId, function(event, data) {
                 _this.openAnnotationList(data);
             });
+
```
Comment (aeschylus) [inline js/src/widgets/editorPanel.js]: We might want to create a new events listener here for the page turn in order to clear or invalidate the rendering.

**Comment by sdellis** [comment]
Ok, thanks for the tips.  Let me see what I can do on this front tomorrow.

**Pull request merged**